cli/runner.go

```diff
@@ -77,6 +78,13 @@ func (c *ServiceRunnerCommand) Execute(_ []string) error {
 		return runProcessByName(ctx, c.Single, settings)
 	}
 
+	pprofHandler := service.GetHandlerPprof(settings)
+	if settings.PprofPort != "" {
+		go func() {
+			err = service.RunGracefully(settings.PprofPort, requestTimeout, pprofHandler)
```

Comment:
grip.Alert(service.RunGracefully()) 

(we're not doing anything useful with this error message.